<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Florida plates</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>361</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayoutMain">
    <property name="spacing">
     <number>15</number>
    </property>
    <property name="leftMargin">
     <number>25</number>
    </property>
    <property name="topMargin">
     <number>25</number>
    </property>
    <property name="rightMargin">
     <number>25</number>
    </property>
    <property name="bottomMargin">
     <number>25</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="hLayoutNumber">
      <item>
       <widget class="QLabel" name="labelNumber">
        <property name="text">
         <string>Количество изображений</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="numberOfRendering">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100000</number>
        </property>
        <property name="singleStep">
         <number>100</number>
        </property>
        <property name="stepType">
         <enum>QAbstractSpinBox::DefaultStepType</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="hLayoutSavePath">
      <item>
       <widget class="QLabel" name="labelSavePath">
        <property name="text">
         <string>Путь сохранения</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="savePath"/>
      </item>
      <item>
       <widget class="QPushButton" name="buttonChoseSavePath">
        <property name="text">
         <string>Выбрать</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="hLayoutFileName">
      <item>
       <widget class="QLabel" name="labelFileName">
        <property name="text">
         <string>Название файла</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="saveFileName">
        <property name="text">
         <string>plate_</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="hLayoutButtonPanel">
      <item>
       <widget class="QPushButton" name="buttonGenerate">
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="toolTipDuration">
         <number>6</number>
        </property>
        <property name="text">
         <string>Сгенерировать</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonStopRender">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Остановить</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>true</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="format">
       <string>%p%</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
